src/components/Inputs/Technoeconomic/TechnoeconomicInputs.tsx

```diff
@@ -25,7 +19,34 @@ interface Props {
   disabled: boolean;
 }
 
+const determineScaledCapitalCost = (electricalCapacity: number) => {
+  const scaleFactor = 0.82; // source: https://evcvaluation.com/cost-to-capacity-method-applications-and-considerations/
+  const defaultCapitalCost = 70_000_000;
+  const defaultElectricCapacity = 25_000;
+
+  if (electricalCapacity === defaultElectricCapacity) {
+    return defaultCapitalCost;
+  } else {
+    const scaleCapitalCost =
+      (defaultCapitalCost * (electricalCapacity / defaultElectricCapacity)) ^
+      scaleFactor;
+    return scaleCapitalCost;
+  }
+};
+
 export const TechnoeconomicInputs = (props: Props) => {
+  useEffect(() => {
+    const scaledCapitalCost = determineScaledCapitalCost(
+      props.teaInputs.ElectricalFuelBaseYear.NetElectricalCapacity
+    );
+
+    if (scaledCapitalCost !== props.teaInputs.CapitalCost) {
```

Comment:
we should check if `CapitalCost` is specified by users; if it is, we will not want to scale/modify it. 
only when `CapitalCost` is not specified by users, should we scale the default CapitalCost using `NetElectricalCapacity`.